contents/en/basics/inspiration.md

```diff
@@ -4,3 +4,4 @@ The following list provides some ideas when you have a question "I don't know ho
 
 - [Apache ECharts Official Examples](${mainSitePath}/examples)
 - "How To Guides - Common Charts" in this handbook
+- Works from makeapie.com community 
```

Comment:
It should contain the link as Chinese version does. Like `Community works from [makeapie.com](https://www.makeapie.com/)`.

Reply:
Noted，this is my first time to contribution. Do I need to open a new PR or some better way？

Reply:
@dbgee You may click [here](https://github.com/dbgee/echarts-handbook/edit/patch-2/contents/en/basics/inspiration.md?pr=%2Fapache%2Fecharts-handbook%2Fpull%2F28
) to continue to make changes and commit to the current branch `patch-2`.